src/openapi.json

```diff
@@ -7,11 +7,21 @@
   },
   "components": {
     "securitySchemes": {
-      "preSharedApiKey": {
-        "type": "apiKey",
-        "description": "A pre-shared key required to create or access protected resources via the API.",
-        "name": "x-api-key",
-        "in": "header"
+      "auth": {
+        "type": "oauth2",
```

Comment:
I tried for a while to get this to work with `"type": "openIdConnect",` as the [docs say it should](https://swagger.io/docs/specification/authentication/openid-connect-discovery/), but I wasn't able to before I reached the end of the timebox I set for myself. :disappointed:

Reply:
I was able to use `"type": "openIdConnect"` locally. The user experience was not good because an exhaustive litany of capabilities or options showed up. With the `oauth2` type, I was able to present exactly one known-good option.

Reply:
Fair enough! Thanks for following up!